GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+# Governance
+
+## Glossary
+
+* **Advisory Committee (AC).**  A group of people with representation from a variety of AMP's constituencies including Users, End-users and Collaborators who provide advice to the Technical Steering Committee.
+
+* **Collaborators.**  People who have been granted write-access to ampproject repositories.
```

Comment:
@cramforce can we clarify here if collaborators with have merge access? (since a person can be a github collaborator but not part of the amphtml master mergers group which you need to be in to have merge access)